man/systemd-timesyncd.service.xml

```diff
@@ -64,6 +64,12 @@
     reboots to ensure it monotonically advances even if the system
     lacks a battery-buffered RTC chip.</para>
 
+    <para>The <filename>systemd-timesyncd</filename> service
+    specifically implements only SNTP. This minimalistic
+    service will generally set the system clock rather
+    than slowly adjust it. More complex use cases are not
+    covered by <filename>systemd-timesyncd</filename>.</para>
```

Comment:
That's not true actually. Please see `manager_adjust_clock` — the kernel should slew the clock for adjustments smaller than 0.5 s.